project/config.py

```diff
@@ -7,3 +7,7 @@ class Config(object):  # pylint: disable=too-few-public-methods
     """Default Flask configuration."""
     DEBUG = True
     SECRET_KEY = os.environ.get('SECRET_KEY') or 'test-key'
```

Comment:
is this line redundant now?

Reply:
Oh SECRET_KEY is lowkey used by Flask for CSRF protection. Added a comment.